src/sentry/interfaces/schemas.py

```diff
@@ -156,6 +156,64 @@ def apierror(message="Invalid data"):
     'additionalProperties': {'not': {}},
 }
 
+EXCEPTION_MECHANISM_INTERFACE_SCHEMA = {
```

Comment:
The schema for mechanism is declared here. I was a bit lazy to allow `null` values in each case (e.g. for `code_name`) and prune empty keys in the interface instead. Note that `meta` is "strongly typed" (and the interface will throw away all unknown properties), whereas `data` is not.